tests/eval-one.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+set -u
+here=$(cd "$(dirname "$0")" && pwd)
+
+if [ -z "$1" ]; then
+  echo "Usage: $0 <profile>"
+  exit 1
+fi
+
+profile=$1
+logfile=$(mktemp)
+
+# shellcheck disable=SC2039
+echo -n "evaluating $profile: "
+
+nix-build \
+  '<nixpkgs/nixos>' \
+  -I "nixos-config=$here/eval-test.nix" \
+  -I "nixos-hardware-profile=$profile" \
```

Comment:
We shouldn't allow profiles that have spaces in names.

Reply:
It doesn't hurt to escape the argument in either case

Reply:
It does. We should enforce that there is no white spaces in names.